### DISCUSSION TITLE
Unable to Sign in to Copilot

### DISCUSSION THREAD

**Discussion by ablacklama** [community/community#19103] [Copilot Conversations]
I've been using copilot successfully for a while now, but after getting the email today it's stopped working. 
![image](https://user-images.githubusercontent.com/22246943/174906393-0c0a72a1-c2fe-4897-8da2-07cf047b98f4.png)
When try to sign in and pay for a subscription I get to these pages:
![image](https://user-images.githubusercontent.com/22246943/174906475-0cedc719-b263-4de2-b680-d23344397d20.png)
![image](https://user-images.githubusercontent.com/22246943/174906515-402c79a1-0812-42da-8ffd-7c18ea259858.png)
But as you can see, there's no way for me to actually pay for the subscription.
Then I go back to the first page and click "continue without payment details". I set my copilot preferences, and click "save and get started". This seems to work and it says "GitHub Copilot is now ready to pair". But nothing's changed. When i go to my billing page it still says that i need to activate my trial. 
![image](https://user-images.githubusercontent.com/22246943/174906784-da88b013-344e-4489-bfbd-9a78dce8721e.png)
And reloading vscode bring back the "no access to github copilot found." message.

EDIT UPDATE:
It's giving me a new message now.
![image](https://user-images.githubusercontent.com/22246943/174931940-f62a3f27-8db5-4ae2-b325-3b91879b37f3.png)
So it looks like my account got locked from purchases back in early 2021. I'm not sure why. I updated my payment method and that didn't work, so i've created a support ticket.

**Comment by Jropp** [comment]
I'm seeing similar issues but this is my Visual Studio Code message 
![Screen Shot 2022-06-21 at 6 18 45 PM](https://user-images.githubusercontent.com/23724997/174907089-dcd9eaa9-d5f4-4739-b1fb-07e94f54fd3f.png)

Also I don't have the "continue without payment details" option, perhaps because I already have a payment method attached?

**Reply by superAo005** [reply]
Also I don't have the "continue without payment details" option why？

**Comment by ablacklama** [comment]
It's giving me a new message now.
![image](https://user-images.githubusercontent.com/22246943/174925598-c19a83b1-18ec-4341-a62f-ac95dcf09c06.png)
So it looks like my account got locked from purchases back in early 2021. I'm not sure why. I updated my payment method and that didn't work, so i've created a support ticket.

**Reply by ShinChven** [reply]
I'm getting this too.

**Reply by shd101wyy** [reply]
Yep I am getting the same error :)

**Reply by pyronaur** [reply]
~Seeing the same issue~

No longer an issue for me. I guess caches have updated?

**Reply by mogzol** [reply]
I was having this same problem, and I'm not sure what I did, but some combination of changing my payment method back and forth between paypal and credit card, and updating my billing address finally got this message to go away and I was able to activate copilot.

**Comment by rauzerfabiano** [comment]
1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor

**Comment by galah92** [comment]
Same thing for me

**Reply by rauzerfabiano** [reply]
1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor

**Reply by ShinChven** [reply]
it didn't solve the problem.

**Reply by superAo005** [reply]
Also I don't have the "continue without payment details" option why？

**Reply by ShinChven** [reply]
<img width="645" alt="image" src="https://user-images.githubusercontent.com/3351486/174936243-93a76728-e369-4a90-8801-f2a781716175.png">

**Reply by superAo005** [reply]
![image](https://user-images.githubusercontent.com/22334521/174936467-ebcac99e-4f19-4fff-b8b5-b2f13956d0e7.png)
it is my

**Comment by mashirozx** [comment]
Same, vscode
<img width="461" alt="image" src="https://user-images.githubusercontent.com/16148054/174943473-caf03bae-95bd-4586-a1b5-b3228ea91062.png">
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/16148054/174944347-5e631e8b-94bb-48ba-855e-58d81e36de05.png">

<img width="795" alt="image" src="https://user-images.githubusercontent.com/16148054/174943497-c756a3bb-0b59-41cb-83d2-78e354a413b8.png">

**Reply by JuanDavidBallesteros** [reply]
Hi @mashirozx,

I'm facing the same issue, but I can't find how to solve it. Did you find a solution?

**Reply by mashirozx** [reply]
resolved by their stuff，please contact GitHub support https://support.github.com/
![Screenshot_2022-09-09-00-46-59-875_com kiwibrowser browser](https://user-images.githubusercontent.com/16148054/189179953-d91f8c5e-2191-429f-91be-23f5be1c0f3c.jpg)

**Comment by mashirozx** [comment]
Seems the web auth doesn't redirect to the auth callback URL correctly, which redirect to the bill page incorrectly.

**Reply by bzctoons** [reply]
Hi, same for me

**Reply by davecheney** [reply]
@bzctoons i'm sorry you are having difficulties. Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by jeeva-Bharathi-mac** [reply]
Same for me as well. I signed out in browser, went back to vscode, tried to initiate signin from vscode, after signing in the browser it redirects to payment and billing

**Reply by mashirozx** [reply]
@jeeva-Bharathi-mac https://github.com/orgs/community/discussions/19103#discussioncomment-3597446

**Reply by jeeva-Bharathi-mac** [reply]
@mashirozx Thank you

**Comment by conciertos** [comment]
I don't get the notification sigh. I only get a window that redirects me here https://github.com/settings/copilot
